src/webjack.js

```diff
@@ -12,7 +12,8 @@ WebJack.Connection = Class.extend({
     }
 
     var args = ifUndef(args, WebJack.Profiles.SoftModem);
-    var audioCtx = typeof args.audioCtx === 'undefined' ? new AudioContext() : args.audioCtx;
+    var AudioContext = window.AudioContext || window.webkitAudioContext;
```

Comment:
Oh no, this line broke the tests... let me look for how `window` is referenced in the tape testing context... hang on...

Reply:
OK... something like `if(typeof(window) !== "undefined") {`… ?

Reply:
Could you try `var AudioContext = AudioContext || webkitAudioContext;`?